modules/luci-base/htdocs/luci-static/resources/validation.js

```diff
@@ -337,7 +337,7 @@ var ValidatorFactory = baseclass.extend({
 		},
 
 		host: function(ipv4only) {
-			return this.assert(this.apply('hostname') || this.apply(ipv4only == 1 ? 'ip4addr' : 'ipaddr'),
```

Comment:
Why not simply `... || this.apply(ipv4only == 1 ? 'ip4addr' : 'ipaddr', null, ['nomask']),`  ?

Reply:
Thanks for the advice. I think I was blind!